pkgs/top-level/python-packages.nix

```
@@ -628,6 +628,8 @@ in {
 
   ase = callPackage ../development/python-modules/ase { };
 
+  asf_search = callPackage ../development/python-modules/asf_search { };
```

```suggestion
  asf-search = callPackage ../development/python-modules/asf-search { };
```
Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)